bin/shell

```diff
@@ -16,19 +16,22 @@ end
 
 ecs_client = Aws::ECS::Client.new(region: 'us-west-2')
 ec2_client = Aws::EC2::Client.new(region: 'us-west-2')
+cluster = if env == "sandbox"
+            "idseq-#{env}-ecs"
+          else
+            # TODO(2020-01-23): Rename staging/prod clusters to new format (IDSEQ-2155).
+            env
+          end
 service = "idseq-#{env}-web"
 
-resp = ecs_client.describe_services(cluster: env, services: [service])
-service = resp.services.first
```

Comment:
Lines 21 and 22 here weren't doing anything anymore (you'll see `service` variable is not used later)